docs/insforge-instructions-sdk.md

```
@@ -12,15 +12,16 @@ Backend-as-a-service (BaaS) platform providing:
 
 ## Installation
 
+**🚨 CRITICAL: Follow these steps in order**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)